Mock-Get-Started/Mock-Get-Started-MyApp/src/main/java/com/microej/example/mock/MockUsageDemo.java

```diff
@@ -49,60 +46,53 @@ public static void main(String[] args) {
 	public static void start() {
 		// Start MicroUI framework.
 		MicroUI.start();
+		MySNI.doNothing();
+		showMainPage();
 
-		// Initialize stylesheet rules.
-		StylesheetPopulator.initialize();
-
-		// Create the navigator.
-		TransitionContainer = new SlideTransitionContainer(MWT.LEFT, false);
-
-		// Show the main page.
-		mainPage = new MyPage();
-		TransitionContainer.show(mainPage, false);
-
-		keyboardPage = new KeyboardPage();
-
-		// Show the navigator.
-		Desktop desktop = new Desktop() {
-			@Override
-			public boolean handleEvent(int event) {
-				// set panel focus to null when we click on a blank space
-				int type = Event.getType(event);
-				if (type == Event.POINTER) {
-					int action = Pointer.getAction(event);
-					if (action == Pointer.RELEASED) {
-						getPanel().setFocus(null);
-					}
-				}
-				return super.handleEvent(event);
-			}
-		};
-		Panel = new Panel();
-		Panel.setWidget(TransitionContainer);
-		Panel.showFullScreen(desktop);
-		desktop.show();
-	}
 
-	/**
-	 * Gets the panel.
-	 *
-	 * @return the panel.
-	 */
-	public static Panel getPanel() {
-		return Panel;
 	}
 
 	/**
 	 * Shows the main page of the application.
 	 */
 	public static void showMainPage() {
-		TransitionContainer.show(mainPage, false);
+		Desktop desktop = createDesktop(new MyPage());
+		StylesheetPopulator.initialize(desktop);
+		displayPage(desktop);
 	}
 
+
 	/**
-	 * Shows the keyboard page.
+	 * Show keyboard page.
 	 */
 	public static void showKeyboardPage() {
-		TransitionContainer.show(keyboardPage, true);
+		KeyboardPage keyboardpage = new KeyboardPage();
+		Desktop desktop = createDesktop(keyboardpage.getContentWidget());
+		displayPage(desktop);
+	}
+
+	/**
+	 * Display page.
+	 *
+	 * @param desktop the desktop
+	 */
+	public static void displayPage(Desktop desktop) {
+		StylesheetPopulator.initialize(desktop);
+		TransitionDisplayable displayable = new TransitionDisplayable(desktop, true);
+		Display.getDisplay().requestShow(displayable);
+	}
+
+	/**
+	 * Creates the desktop.
+	 *
+	 * @param page the page
+	 * @return the desktop
+	 */
```

Comment:
just saying the page or the desktop is not enough, you should say meaningful stuff, like:
- what you're going to do with it;
- what is method good for;
- when you should use it;
- etc

For example:

> Creates a desktop with the given page as the root widget.

Frankly, I don't even know why this method is needed, you should just remove it.